src/main/java/cli/scrape.java

```diff
@@ -0,0 +1,236 @@
+package cli;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.swagger.v3.core.util.Json;
+import models.SubjectCode;
+import models.Term;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+import scraping.QuerySubject;
+import scraping.ScrapeCatalog;
+import scraping.ScrapeSection;
+import services.JsonMapper;
+import services.ParseSchoolSubjects;
+import services.Utils;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+@CommandLine.Command(name = "scrape")
+public class scrape {
+    public static class Section implements Runnable {
+        private Logger logger = LoggerFactory.getLogger("scrape.section");
+
+        @CommandLine.Option(names = "--term", required = true)
+        private String term;
+
+        @CommandLine.Option(names = "--registrationNumber", required = true)
+        private String registrationNumber;
+
+        @CommandLine.Option(names = "--outputFile")
+        private String outputFile;
+
+        @CommandLine.Option(names = "--pretty")
+        private String pretty;
+
+        public void run() {
+            long start = System.nanoTime();
+            try {
+                Utils.writeToFileOrStdOut(
+                        JsonMapper.toJson(
+                                ScrapeSection.scrapeFromSection(Term.fromId(Integer.parseInt(term))
+                                        , Integer.parseInt(registrationNumber))
+                        ),outputFile);
+            } catch (IOException e) {
+                e.printStackTrace();
+            } catch (ExecutionException e) {
+                e.printStackTrace();
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+            long end = System.nanoTime();
+            double duration = (end - start) /  1000000000.0;
+            logger.info(duration + "seconds");
+        }
+    }
+
+    public static class Sections implements Runnable {
+        private Logger logger = LoggerFactory.getLogger("scrape.sections");
+
+        @CommandLine.Option(names = "--term", required = true)
+        private String term;
+
+        @CommandLine.Option(names = "--school")
+        private String school;
+
+        @CommandLine.Option(names = "--subject")
+        private String subject;
+
+        @CommandLine.Option(names = "--outputFile")
+        private String outputFile;
+
+        @CommandLine.Option(names = "--batchSize")
+        private String batchSize;
+
+        @CommandLine.Option(names = "--pretty")
+        private String pretty;
+
+        public void run() {
+            long start = System.nanoTime();
+            String school = this.school;
+            String subject = this.subject;
+            Term term = Term.fromId(Integer.parseInt(this.term));
+            if(school != null && subject != null) {
```

Comment:
Maybe we could run the code through an autoformatter
https://github.com/google/google-java-format?